policyengine_canada/variables/gov/provinces/bc/benefits/bcfb/bc_family_benefit_first_reduction.py

```diff
@@ -0,0 +1,42 @@
+from policyengine_canada.model_api import *
+
+
+class bc_family_benefit_first_reduction(Variable):
+    value_type = float
+    entity = Household
+    label = "British Columbia family benefit first reduction"
+    unit = CAD
+    definition_period = YEAR
+
+    def formula(household, period, parameters):
+        income = household("adjusted_family_net_income", period)
+        children = household("bc_family_benefit_eligible_children", period)
+        p = parameters(period).gov.provinces.bc.benefits.bcfb.first_reduction
+        base = household("bc_family_benefit_base", period)
+        reduction = p.rate.calc(income)
+        reduction_cap = (
+            ((children > 0) * p.max_amount.one_child)
+            + ((children > 1) * p.max_amount.two_children)
+            + (max_(children - 2, 0) * p.max_amount.three_or_more_children)
+        )
+        print(reduction_cap)
```

Comment:
remove this and commented code below. could you also add some comments, given this is a nontrivial computation?